src/main/java/com/raisetech/mybatisdemo20236/controller/TwitterController.java

```diff
@@ -0,0 +1,93 @@
+package com.raisetech.mybatisdemo20236.controller;
+
+import com.raisetech.mybatisdemo20236.entity.Twitter;
+import com.raisetech.mybatisdemo20236.exception.ResourceNotFoundException;
+import com.raisetech.mybatisdemo20236.form.TwitterCreateForm;
+import com.raisetech.mybatisdemo20236.form.TwitterUpdateForm;
+import com.raisetech.mybatisdemo20236.service.TwitterService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class TwitterController {
+
+    private final TwitterService twitterService;
+
+    public TwitterController(TwitterService twitterService) {
+        this.twitterService = twitterService;
+    }
+
+    @GetMapping("/twitter/{id}")
```

Comment:
ツイート検索なら、
```suggestion
    @GetMapping("/tweets/{id}")
```
でも良いかなと思いました。

Reply:
ご助言ありがとうございます。
修正してみます。